apps/api/v2/src/modules/organizations/organizations.repository.ts

```diff
@@ -16,6 +16,7 @@ export class OrganizationsRepository {
     return this.dbRead.prisma.team.findUnique({
       where: {
         id: organizationId,
+        isOrganization: true,
```

Comment:
Since this is the organizations repository when we're retrieving organizations we should filter for the `isOrganization` field.

Reply:
Well spotted!